autoload/lsp/omni.vim

```diff
@@ -270,6 +270,14 @@ endfunction
 " @return { 'items': v:completed_item[], 'incomplete': v:t_bool, 'startcol': number }
 "
 function! lsp#omni#get_vim_completion_items(options) abort
+    if lsp#utils#has_vim9script() && g:lsp_use_vim9script
+        return s:GetVimCompletionItems(a:options)
+    else
+        return s:get_vim_completion_items(a:options)
+    endif
+endfunction
```

Comment:
The official documentation describe as following
```
There is one way to use both legacy and Vim9 syntax in one script file:
        " comments may go here
        if !has('vim9script')
           " legacy script commands go here
           finish
        endif
        vim9script
        # Vim9 script commands go here
```
https://github.com/vim/vim/blob/a7d36b732070944aab614944075ec0b409311482/runtime/doc/vim9.txt#L1719

The style have a problem that it needs to indent legacy scripts. Currently, almost program is written in `vim-lsp` and it will be the same way in the future.

The style of my changes is based on existing code written in lua.

https://github.com/prabirshrestha/vim-lsp/blob/25337623cf11e2ac9c73dcba1a48b86fe103d887/autoload/lsp/utils/diff.vim#L14-L44

Which style do you like?